stdlib/functools.pyi

```diff
@@ -131,3 +132,19 @@ if sys.version_info >= (3, 8):
 
 if sys.version_info >= (3, 9):
     def cache(__user_function: Callable[..., _T]) -> _lru_cache_wrapper[_T]: ...
+
+class _HashedSeq(Generic[_T], list[_T]):
```

Comment:
Seems like we will need to `from typing import List` after all, because this file needs to support Python 2:

```suggestion
class _HashedSeq(Generic[_T], List[_T]):
```